okta/models/token_authorization_server_policy_rule_action_inline_hook.py

```diff
@@ -0,0 +1,45 @@
+# flake8: noqa
+"""
+Copyright 2020 - Present Okta, Inc.
```

Comment:
This is a new model and the header should be `2021 - Present Okta, Inc`.

Reply:
I see... I guess, I need to update template/generator first.